### DISCUSSION TITLE
Layered tabs?

### DISCUSSION THREAD

**Discussion by King-Spartacus** [pulse-browser/browser#236] [Ideas]
I’m not known for using many “advanced” features in browsers, but one I’ve recently really grown to like is Vivaldi’s layered tabs. You can group tabs either in a custom way or quickly based on host. This has been helpful to me especially since returning to RuneScape where I can have just “one” OSRS Wiki tab, but upon clicking it a whole second row under the main one with all of my grouped tabs appears.

**Comment by ghost** [comment]
Hell yes. Tab stacks and workspaces are why I use Vivaldi over Firefox. It's nice to compartmentalise groups of tabs by hosts into stacks, and then the types of tabs (e.g. social media, music, video, projects) into workspaces.
